### PR TITLE
cmd ref: dvc add 1.0 update

### PR DESCRIPTION
**Pull request by jorgeorpinel**
> Steals some of Ruslan's work from #1387

- [x] Change the PR target to `master` once #1382 is merged.

> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

File: content/docs/command-reference/add.md [hunk 1]
```diff
@@ -6,11 +6,11 @@ Track data files or directories with DVC, by creating a corresponding
...
 
 ```usage
-usage: dvc add [-h] [-q | -v] [-R] [--no-commit] [-f <filename>]
-               targets [targets ...]
+usage: dvc add [-h] [-q | -v] [-R] [--no-commit] [--external]
+               [-f <filename>] targets [targets ...]
 
 positional arguments:
-  targets               Input files/directories to add.
+  targets               Files or directories to add
```
Comment (jorgeorpinel) [inline content/docs/command-reference/add.md:13]: Needs core repo change.

File: content/docs/command-reference/add.md [hunk 2]
```diff
@@ -139,32 +143,24 @@ $ tree
...
 
-This is a standard `.dvc` file with only one output (in the `outs` field). The
-hash value should correspond to a file path in the <abbr>cache</abbr>.
+This is a standard `.dvc` file with only one output (`outs` field). The hash
+value (`md5` field) corresponds to a file path in the <abbr>cache</abbr>.
 
-> Note that the `meta` values above were entered manually for this example. Meta
-> values and `#` comments are not preserved when a `.dvc` file is overwritten
+> Note that `#` comments are not preserved when a `.dvc` file is overwritten
 > with the `dvc add`, `dvc run`, `dvc import`, or `dvc import-url` commands.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/add.md:155]: Should we just remove all the info. about YAML comments from this doc? Or at least try to move it to another example and not the first one?

Comment (shcheklein) [reply content/docs/command-reference/add.md:155]: where does this question come from? did we change any logic?

Comment (jorgeorpinel) [reply content/docs/command-reference/add.md:155]: There's no longer `meta` AFAIK. I've been removing that.

As for comments, those are still supported but explained in the DVC file guide. I'm saying the add cmd ref may not be the best place to even mention them, especially not in the most basic example.

Comment (shcheklein) [reply content/docs/command-reference/add.md:155]: no meta only for .dvc files, right?

File: content/docs/command-reference/add.md (hunk 1 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/add.md:13]: Mah we'll probably need to check all the commands to match usage and options from docs to core after this whole 1.0 drive... Created iterative/dvc/issues/4024

File: content/docs/command-reference/add.md (hunk 2 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/add.md]: I decided to remove it in 9bd9962b. It's explained in the files guide, which is linked to several times form this doc.

File: content/docs/command-reference/add.md [hunk 3]
```diff
@@ -6,11 +6,11 @@ Track data files or directories with DVC, by creating a corresponding
 ## Synopsis
 
 ```usage
-usage: dvc add [-h] [-q | -v] [-R] [--no-commit] [-f <filename>]
-               targets [targets ...]
+usage: dvc add [-h] [-q | -v] [-R] [--no-commit] [--external]
+               [-f <filename>] targets [targets ...]
```
Comment (shcheklein) [inline content/docs/command-reference/add.md:10]: this has changed in 1.0 - it's --file now

Comment (jorgeorpinel) [reply content/docs/command-reference/add.md:10]: Good catch, will update...

**Comment by jorgeorpinel** [comment]
> name of a PR - quite a few changes here, but almost all of them are not related to 1.0 (and we actually didn't include some 1.0 related changes we made to dvc add at the same time)

All the examples were reviewed with 1.0 and I also reviewed the whole description to see it's still current.

Other than -f did you notice any other 1.0 change I missed? Thanks

**Comment by shcheklein** [comment]
> Other than -f did you notice any other 1.0 change I missed? Thanks

don't remember from the top of my head, the easiest way is to run `dvc add --help` :)

**Comment by jorgeorpinel** [comment]
-f wasn't included because it wasn't there when I did this changes, that PR was merged 2 days ago no docs issue/PR was started by core team: iterative/dvc/pull/4004

I did notice this and left a message in the PR

File: content/docs/command-reference/add.md (hunk 3 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/add.md:10]: Extracted to https://github.com/iterative/dvc.org/issues/1426, please assign a priority - very recent change.

**Comment by jorgeorpinel** [comment]
I ran -h at the time of making this PR of course. There's lots of ongoing core changes and it's not clear to me where the 1.0 release ends and the regular updates begin TBH it's causing some confusion e.g. https://github.com/iterative/dvc/pull/4011#issuecomment-642785573

**Comment by shcheklein** [comment]
> I ran -h at the time of making this PR of course.

I know. I mean that we can run `-h` for all commands again as part of the DVC 1.0 migration. That's what that epic for. It's not a goal to solve all of these discrepancies at once or even before the release - we need to focus on major things.

**Comment by jorgeorpinel** [comment]
Got it. Good idea, added a checkbox to the epic and created iterative/dvc/issues/4024. For now this specific matter (add -f -> --file) is extracted to #1426

**Pull request merged**